			   READLINE PATCH REPORT
			   =====================

Readline-Release: 6.1
Patch-ID: readline61-002

Bug-Reported-by:	Chet Ramey <chet.ramey@case.edu>
Bug-Reference-ID:
Bug-Reference-URL:

Bug-Description:

The readline version information was not updated for the release of version 6.1.

--- ./patchlevel.orig	2010-04-10 12:21:04.611086065 +0000
+++ ./patchlevel	2010-04-10 12:23:12.441086084 +0000
@@ -1,3 +1,3 @@
 # Do not edit -- exists only for use by patch
 
-1
+2
--- ./readline.h.orig	2010-04-10 12:20:48.591085790 +0000
+++ ./readline.h	2010-04-10 12:23:12.441086084 +0000
@@ -39,9 +39,9 @@
 #endif
 
 /* Hex-encoded Readline version number. */
-#define RL_READLINE_VERSION	0x0600		/* Readline 6.0 */
+#define RL_READLINE_VERSION	0x0601		/* Readline 6.1 */
 #define RL_VERSION_MAJOR	6
-#define RL_VERSION_MINOR	0
+#define RL_VERSION_MINOR	1
 
 /* Readline data structures. */
 
